Common/Log.h

```diff
@@ -91,7 +91,7 @@ void GenericLog(LogTypes::LOG_LEVELS level, LogTypes::LOG_TYPE type,
 bool GenericLogEnabled(LogTypes::LOG_LEVELS level, LogTypes::LOG_TYPE type);
 
 #if defined(LOGGING) || defined(_DEBUG) || defined(DEBUGFAST) || defined(_WIN32)
-#define MAX_LOGLEVEL DEBUG_LEVEL
+#define MAX_LOGLEVEL VERBOSE_LEVEL
```

Comment:
Not sure if we want to change this?

-[Unknown]

Reply:
I couldn't see any verbose log when debugging without it, any better way?

Reply:
btw @hrydgard / @unknownbrackets is there anyway we can see Verbose logs in PPSSPP Debug Console Log without changing the source code like this? 

Because changing the Logging Channel of SCENET to "Verb" from PPSSPP Settings doesn't seems to be working :( (was it a bug?)
And changing the source code + rebuild whenever we want to see verbose log is kinda troublesome isn't?